pages/migrations/seed.mdx

```diff
@@ -1,5 +1,36 @@
 # Seed migration
 
+## From `0.85.0` and above
+
+### `new SnapletClient()` -> `await createSeedClient()`
```

Comment:
@jgoux would you like to add your breaking changes above here? (but still underneath `From `0.85.0` and above`)

Reply:
yep I'll add my part